Remove personal entity warnings from Inference docs (#1845)
wandb/docs: weave/cookbooks.mdx

---
title: "Overview"
description: "Practical tutorials and examples for building with Weights & Biases Weave"
mode: wide
---

> Welcome to the Weights & Biases Weave tutorial collection. These hands-on tutorials provide practical examples and best practices for building applications with Weave.

Reviewer: ```suggestion
Welcome to the W&B Weave tutorial collection. These hands-on tutorials provide practical examples and best practices for building applications with Weave.
```

> This section of the docs differs from the Guides section in that these tutorials combine concepts across the Weave's functionality to steer you towards a specific end goal, as opposed to documenting specific pieces functionality.

Reviewer: ```suggestion
This section of the documentation complements the [Guides](/weave/guides) section. Tutorials combine multiple Weave concepts to achieve a specific end goal.
```
It looks like Guides could use a landing page / overview at the top.
